packages/runner/src/lib/SecScan.ts

```diff
@@ -0,0 +1,47 @@
+import { Configuration } from '@secbox/core';
+import { Reporter } from '@secbox/reporter';
+import {
+  ScanFactory,
+  ScanSettingsOptions,
+  Severity,
+  TargetOptions
+} from '@secbox/scan';
+
+export class SecScan {
+  private readonly scanFactory: ScanFactory;
+  private readonly reporter: Reporter;
+  private _threshold = Severity.LOW;
+
+  constructor(
+    private readonly configuration: Configuration,
+    private readonly settings: Omit<ScanSettingsOptions, 'target'>
+  ) {
+    this.scanFactory = this.configuration.container.resolve(ScanFactory);
+    this.reporter = this.configuration.container.resolve(Reporter);
+  }
+
+  public async run(target: TargetOptions): Promise<void> {
+    const scan = await this.scanFactory.createScan({
+      ...this.settings,
+      target
+    });
+
+    try {
+      await scan.expect(this._threshold || (() => true));
```

Comment:
What exactly did you want to achieve by passing `() => true`? It makes `expect()` return control right after the first interaction. 

I suggest passing `Severity.LOW` by default https://github.com/NeuraLegion/sec-tester-js/blob/4710ea6af30335131d8501632f3340bf9dcf8d00/packages/scan/src/models/Severity.ts#L7-L17

```suggestion
      await scan.expect(this._threshold || Severity.LOW);
```

Reply:
indeed this is rudiment place as `_threshold`  was initially `Severity | undefined`. But now `_threshold` is `Severity` with `Severity.LOW` as default and that code is unreachable